README.md

```diff
@@ -6,52 +6,39 @@
 ---
 [![BoaConstrictor](https://img.shields.io/badge/boa%20constrictor%20-The%20.NET%20Screenplay%20Pattern-blueviolet?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA8AAAATCAYAAABPwleqAAAACXBIWXMAAAWJAAAFiQFtaJ36AAABAUlEQVQ4jY2T623CQBCEP6L8xyW4A6ACKCEdxCWEDkgHUAHQQTpISnA6IB2YCjYatIeO5fwY6WTt+eY8s7PGzErrZGVczKxJ50vE2sy6HnKCLmd2u+EZS2AfditgkdXbSK6cGNEBLdAAR3/3l6RWZrYfkZq83i0lcjtCND9DvvEK7IKXX+DHpeZQDx4sybMOzb0+AB893uX3PavPBGkb9z9mQ77rSJ6a81fKOc9q5ZFI8luIcBN68ynyKfNy9Qa2U3OeItOGcl760A9B7x96FMezcb8xJsmWPT2/85wLo/yUs2rlv445xzU555eer9YlOQ6Nr2K7DP3Pec4JmnktAP4BJn0sgWz0+e4AAAAASUVORK5CYII=
 )](https://github.com/q2ebanking/boa-constrictor)
-[![Hacktoberfest](https://img.shields.io/github/hacktoberfest/2021/q2ebanking/boa-constrictor)](https://github.com/q2ebanking/boa-constrictor/issues)
-[![NugetPackage](https://img.shields.io/nuget/v/Boa.Constrictor?label=NuGet%20Package)](https://www.nuget.org/packages/Boa.Constrictor/)
-[![NugetDownloads](https://img.shields.io/nuget/dt/Boa.Constrictor)](https://www.nuget.org/stats/packages/Boa.Constrictor?groupby=Version)
 [![Contributors](https://img.shields.io/github/contributors/q2ebanking/boa-constrictor)](https://github.com/q2ebanking/boa-constrictor/graphs/contributors)
 [![Commits](https://img.shields.io/github/commit-activity/m/q2ebanking/boa-constrictor)](https://github.com/q2ebanking/boa-constrictor/commits/main)
 [![OpenPullRequests](https://img.shields.io/github/issues-pr/q2ebanking/boa-constrictor)](https://github.com/q2ebanking/boa-constrictor/pulls)
 [![OpenIssues](https://img.shields.io/github/issues/q2ebanking/boa-constrictor)](https://github.com/q2ebanking/boa-constrictor/issues)
 [![License](https://img.shields.io/badge/license-Apache%202-blue)](./LICENSE.md)
+[![Hacktoberfest](https://img.shields.io/github/hacktoberfest/2021/q2ebanking/boa-constrictor)](https://github.com/q2ebanking/boa-constrictor/issues)
+
 
 **Boa Constrictor** is the .NET Screenplay Pattern.
```

Comment:
Formatting suggestion: it would be nice if this README followed the pattern you're setting in the other ones. Meaning, the content here could at have a `What is Boa Constrictor?` and a `How do I get started?` headings.

Reply:
Indeed!